src/utils/word-api.ts

```diff
@@ -1,13 +1,26 @@
 /* eslint-disable no-console */
 import { StatefulCitation } from "citeproc";
 
+export type CitationDataFormatForWordAPI = {
+  position: number;
+  citationText: string;
+  citationTag: StatefulCitation;
+};
 class WordApi {
-  static async insertEmptyContentControl(): Promise<unknown> {
+  async insertNewCitation(
+    citations: Array<CitationDataFormatForWordAPI>
```

Comment:
It's still using an array here. Please convert the argument to `citation: CitationDataFormatForWordAPI`